tool-openssl/x509_test.cc

```diff
@@ -276,6 +368,19 @@ class X509ComparisonTest : public ::testing::Test {
   std::string openssl_output_str;
 };
 
+static std::string normalize_subject(std::string input) {
```

Comment:
Can you describe what this is doing or provide some examples, I'm struggling to follow the logic below.

Reply:
We find "subject=" in the provided input. If that string is found, we look for the next newline in input. We extract the substring from "subject=" to the newline. Then we remove all whitespace characters from this string and replace it back into input. 

I've added some documentation for the function